mailer.js

```diff
@@ -11,6 +12,24 @@ var DEFAULT_LOCALE = 'en'
 var DEFAULT_TIMEZONE = 'Etc/UTC'
 
 module.exports = function (log) {
+  // Email template to UTM campaign map
+  var templateNameToCampaignMap = {
+    'passwordResetRequiredEmail': 'password-reset-required',
+    'passwordChangedEmail': 'password-changed-success',
+    'passwordResetEmail': 'password-reset',
+    'postVerifyEmail': 'account-verified',
+    'newDeviceLoginEmail': 'new-device-signin',
+    'recoveryEmail': 'forgot-password',
+    'resetEmail': 'forgot-password',
+    'suspiciousLocationEmail': 'suspicious-location',
+    'verifyEmail': 'welcome',
+    'verifyLoginEmail': 'new-signin',
+    'verificationReminderFirstEmail': 'hello-again',
```

Comment:
For first and second, should we use the same prefix and append a `1`/`2` or `first`/`second` suffix?